src/builtins/filters/object.rs

```diff
@@ -50,4 +59,32 @@ mod tests {
         let result = get(&to_value(&obj).unwrap(), &args);
         assert!(result.is_err());
     }
+
+    #[test]
+    fn test_get_or_null_filter_exists() {
+        let mut obj = HashMap::new();
+        obj.insert("1".to_string(), "first".to_string());
+        obj.insert("2".to_string(), "second".to_string());
+
+        let mut args = HashMap::new();
+        args.insert("key".to_string(), to_value("1").unwrap());
+        args.insert("default".to_string(), Value::Null);
+        let result = get(&to_value(&obj).unwrap(), &args);
+        assert!(result.is_ok());
+        assert_eq!(result.unwrap(), to_value("first").unwrap());
+    }
+
+    #[test]
+    fn test_get_or_null_filter_doesnt_exist() {
+        let mut obj = HashMap::new();
+        obj.insert("1".to_string(), "first".to_string());
+        obj.insert("2".to_string(), "second".to_string());
+
+        let mut args = HashMap::new();
+        args.insert("key".to_string(), to_value("3").unwrap());
+        args.insert("default".to_string(), Value::Null);
```

Comment:
don't use serde_json `Value` in the tests, that's not what's going to happen in templates, especially since we don't have null